### ISSUE TITLE
Update runtime to 3.38

### ISSUE THREAD

**Issue by LeandroStanger** [flathub/com.anydesk.Anydesk#13]

**Comment by Aurnytoraink** [comment]
Well now should be updated to 3.36

**Comment by wjt** [comment]
PR welcome, particularly if you actually use Anydesk and would like to maintain this. https://discourse.flathub.org/t/help-wanted-with-existing-anydesk-flatpak/289

**Comment by Aurnytoraink** [comment]
I will give it a try

**Comment by wberrier** [comment]
And now possibly 3.38...

I got this:

```
$ flatpak update
Looking for updates…
Info: org.gnome.Platform is end-of-life, with reason: The GNOME 3.28 runtime is no longer supported and does not receive security updates. Please ask your application developer to migrate to a supported platform.

...

Error: Can't remove org.gnome.Platform/x86_64/3.28, it is needed for: com.anydesk.Anydesk/x86_64/stable


```

**Comment by wjt** [comment]
If you use Anydesk and would like to adopt this app, a pull request to update it to the 3.38 runtime (or possibly the 20.08 freedesktop runtime?) would be a great start.

**Comment by Inglebard** [comment]
Hi,

I create a pull request #46 about this issue.
If some of you can build it a check if everything is OK.

**Comment by wjt** [comment]
Thanks! I have merged #46.

@Inglebard are you interested in adopting this app on Flathub? If so, I'll ask the Flathub admins to add you to this repo.

**Comment by Inglebard** [comment]
I hope it will works well for everybody.

@wjt , Unfortunately, I think I don't have enough knowledge (about flatpak, github system and compiling application process) to properly maintain this application. 

However since I use it in my company, I need this application to work. So I may in the future create pull requests, if things are not working or outdated and stay in my level of knowledge.

So actually, I'm not interested in adopting this app on Flathub. But thanks for asking me.
